gamemodes/attdef.pwn

```diff
@@ -166,24 +191,76 @@ public OnPlayerRequestClass(playerid, classid)
 	SetPlayerTime(playerid, 12, 0);
 	SetPlayerInterior(playerid, MainInterior);
 	
-	switch(random(3))
+	if(!SightseeingInClassSelection)
```

Comment:
Yes, it is disabled by default. Normally when you are in a class selection, camera moves around the spawn. With this enabled, camera goes around random places around San Andreas map.